site/training/training-overview.qmd

```diff
@@ -0,0 +1,140 @@
+---
+title: "Welcome to ValidMind Academy"
+date: last-modified
+---
+
+Gain hands-on experience and explore what ValidMind has to offer with our training environment. 
+
+The training environment mimics a production environment and includes comprehensive resources such as notebooks with sample code you can run, sample models registered in the model inventory, and draft documentation and validation reports.
+
+::: {.grid}
+
+::: {.g-col-6}
+Most of our model documentation features are available for you to test in the training environment, including automated model testing & documentation, preparing model validation reports, and large language model (LLM) support. 
+
+These features provide a rich context for testing and evaluation, using models and datasets provided by ValidMind and hosted in our own environment. 
+:::
+
+
+::: {.g-col-6}
+![](../assets/img/Group 38-2-compact.png){width=90% fig-alt="An image showing the main ValidMind UI main dashboard"}
+
+:::
+
+:::
+
+
+## Onboarding
+
+Step-by-step instructions: 
+
+1. Get your access credentials from ValidMind. 
+
+   If you haven't reached out to us yet, email <info@validmind.ai> to enquire about access to the training environment.
+
+<!--- FUTURE step for self-signup
+1. Get your access credentials for the {{< var vm_ui >}} by signing up with ValidMind: [{{< var vm_url >}}]({{< var vm_url >}})
+   
+   This is the login URL for our {{< var vm_ui >}} where you can look at the model inventory for the different models. 
+--->
+
+2. When you receive your access credentials, make sure you can log in:
+
+   - ValidMind Jupyter Hub: <https://jupyterhub.validmind.ai/>
+   - {{< var vm_platform >}}: [{{< var vm_url >}}]({{< var vm_url >}})
+
+3. Join a kick-off session with ValidMind to get a free instructor-led hands-on demo.
+
+   After we have received your request to try out the training environment, we will contact you to set up a kick-off session.
+
+4. Try the training environment: [Getting started](#getting-started)
+
+::: {.callout-important}
+## For testing and evaluation purposes only
+
+Do not upload proprietary information — the training environment environment is not to be used for production. If you have questions about testing in the training environment, please [provide feedback or ask for help](#provide-feedback-or-get-help).
```

Comment:
It's because you'd be exposing information that should not be uploaded into a training environment — InfoSec would have a field day. 

One thing missing still is a repeatable, consistent training environment that we can periodically wipe or update. It's coming, along with a dedicated training organization on ValidMind, it's just not quite ready yet. Having that will also allows us to use deep links for many of the `TRY IT` pages that currently take you to the model inventory, e.g. where we tell you to do something with some section of the documentation you already located in an earlier step.